### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by MatissJanis**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!--The content below is only a placeholder and can be replaced.-->
+
+<form #signup="ngForm" (ngSubmit)="seveinfo(signup)">
```
Comment (MatissJanis) [inline src/app/app.component.html]: Very good! Now you only need to add this `seveinfo` method (function) to the `app.component.ts` class. This method should collect all the input data and either
a) log it to console (`console.log(input)`) or
b) make a HTTP request (specific request details will be sent later, so I'd suggest to do console.log for now)

File: src/app/app.component.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        Email is invalid
+        </span>
+
+      </div>
+        <label for tips="ipasumatips" class="control-label">Īpašuma tips</label><br>
+        <mat-radio-group name="ipasumatips" [(ngModel)]="ipasumatips">
+          <mat-radio-button value="dzivokli" checked>Dzīvoklis</mat-radio-button><br>
+          <mat-radio-button value="māju">Māja</mat-radio-button><br>
+          <mat-radio-button value="zemi">Zeme</mat-radio-button><br>
+          <mat-radio-button value="mežu">Mežš</mat-radio-button><br>
```
Comment (MatissJanis) [inline src/app/app.component.html]: It's ok for labels (visual texts) to be in Latvian, but every input value used by the system should be in English.

```suggestion
          <mat-radio-button value="forest">Mežš</mat-radio-button><br>
```

Comment (AlibabaTheThief) [reply src/app/app.component.html]: I agree, but I made so these values show up on next radio buttons when chosen. But there always a better way of doing that...

Comment (MatissJanis) [reply src/app/app.component.html]: The labels will still appear next to the radio buttons in Latvian. What this does (see example above) is set the values in English. These are invisible to people and only visible to machines :)

**Pull request closed without merging**